### DISCUSSION TITLE
use rdf:PlainLiteral instead of rdfs:Literal

### DISCUSSION THREAD

**Discussion by VladimirAlexiev** [eprocurementontology/eprocurementontology#522] [Ideas]
A number of text fields use range `rdfs:Literal`, thus allowing either `xsd:string` or `rdf:langString` (good).
But this also allows datatyped literals, eg numbers or dates (bad).

Instead, use rdf:PlainLiteral, defined at https://w3.org/TR/rdf-plain-literal, which is exactly what's needed.

**Comment by eprocurementontology** [comment]
We will discuss this point when addressing the transformation of the UML to OWL in future evolutions.

**Comment by valexande** [comment]
Dear, @muricna can you give us your opinion on the following comment

We’ve got confirmation from @andreea-pasare  that starting with the next release of EPO (v4.0.0, release candidate to come out in June) all references to rdfs:Literal type will be replaced with rdf:PlainLiteral . 

Since  rdf:PlainLiteral is a subclass of  rdfs:Literal, and neither of them (nor rdf:langString) is explicitly mentioned in the TM, the best course of action would be to update the Master CM to refer to rdf:PlainLiteral instead of rdf:langString. This will be compatible with EPO v.3.1.0 (because of the implicit subtype relationship), but also forward compatible with EPO v.4.0.0. 

we are asking this here, cause is related with optimizing the Conceptual Mapping

**Comment by muricna** [comment]
@valexande your question refers to https://github.com/OP-TED/ted-rdf-mapping please create a ticket in the correct github.  Anyhow the reply is do not change the mappings that are correct for v3.1.0 to which we are mapping.  Future mappings to future versions of the ontology will be discussed when such mappings are necessary.

**Comment by andreea-pasare** [comment]
Resolution to this discussion is mentioned in https://github.com/OP-TED/ePO/issues/405.
Closing this, since the remaining uncertainty is tracked under a different repository: https://github.com/OP-TED/ted-rdf-mapping/issues/407.
